Application/EdFi.Ods.Tests/EdFi.Ods.Api/Controllers/TokenControllerUsingJsonTests.cs

```diff
@@ -25,7 +32,24 @@ namespace EdFi.Ods.Tests.EdFi.Ods.Api.Controllers
     [TestFixture]
     public class TokenControllerUsingJsonTests
     {
-        public class With_valid_key_and_secret : TestFixtureAsyncBase
+        private static ControllerContext CreateControllerContext(HeaderDictionary headerDictionary)
+        {
+            var request = A.Fake<HttpRequest>();
+
+            A.CallTo(() => request.Headers).Returns(headerDictionary);
+
+            var httpContext = A.Fake<HttpContext>();
+            A.CallTo(() => httpContext.Request).Returns(request);
+
+            return new ControllerContext { HttpContext = httpContext };
+        }
+
+        private static string CreateEncodedAuthentication(string clientKey = "clientId", string clientSecret = "clientSecret")
+        {
+            return Convert.ToBase64String(Encoding.UTF8.GetBytes($"{clientKey}:{clientSecret}"));
+        }
```

Comment:
This is duplicated, i think i suggested this should move to the helper class, and is used between both tests

Reply:
Fixed